force-app/main/default/objects/Lead/fields/Area_of_Interest__c.field-meta.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Area_of_Interest__c</fullName>
```

Comment:
Casing issue. You may have to rename all of the files. Casing changes for file names can be challenging to handle with git. I suggest using git mv
```suggestion
    <fullName>Area_Of_Interest__c</fullName>
```